docs/DeveloperGuide.md

```diff
@@ -60,6 +60,26 @@ The figure below represents the sequence diagram when the user wants to add a st
 
 <>
 
+### Model Component
+The `model` package contains the `InstrumentManager` class and the main parent class `Instrument`. It is defined
+in `InstrumentManager.java` and `Instrument.java` respectively. This figure below represents the class diagram of 
+how the different class work together:
+
+<>
+
+The `Model` component:
+
+1. Stores the instrument data through `Instrument` objects which are contained in the `InstrumentManager`
+2. Contains an abstract parent `Instrument` class. The 4 child sub-instrument classes `Crypto`, `Etf`, `Forex` and 
+`Stock` implements the Overridden methods (e.g. toList()).
+3. Contains the `InstrumentManager` which manages the list of instruments (e.g. add a new instrument to the list).
+4. Does not depend on any of the other components (as the `Model` component represents data on the instruments and can 
```

Comment:
Rephrase to like "Does not have any dependencies on any other components, as the instrument is meant to responsible solely for the data representation and modification" 
^something like this